asv_bench/benchmarks/pandas/arithmetic.py

```diff
@@ -0,0 +1,207 @@
+import operator
+
+import numpy as np
+
+import modin.pandas as pd
+from modin.pandas import (
+    DataFrame,
+    Series,
+    date_range,
+)
+
+from .pandas_vb_common import numeric_dtypes
+
+
+class IntFrameWithScalar:
+    params = [
+        [np.float64, np.int64],
+        [2, 3.0, np.int32(4), np.float64(5)],
+        [
+            # subset of operators used in the corresponding pandas asv;
+            #  include floordiv in particular bc it goes through
+            #  its own code-path.
```

Comment:
```suggestion
            # include floordiv in particular bc it goes through
            # its own code-path.
```
Do you mean floordiv goes through its own code-path in pandas?

Reply:
yes